### PR TITLE
enable FT.TAGVALS command

### PR DESCRIPTION
**Pull request by maguec**
add a new function to client which takes the name of a tagfield and runs the following command

```
FT.TAGVAL <INDEX> tagfield
```

Returning a list of tags.

Test is included in test/test.py

### CODE REVIEW

**Review by rafie — APPROVED** [review]

File: test/test.py [hunk 1]
```diff
@@ -617,6 +619,11 @@ def testTags(self):
                 res = client.search(q)
                 self.assertEqual(1, res.total)
 
+                q2 = client.tagvals('tags')
+                self.assertEqual(tags.split(','), q2)
+
+
+
```
Comment (ashtul) [inline test/test.py:626]: remove spaces

File: test/test.py [hunk 2]
```diff
@@ -617,6 +619,11 @@ def testTags(self):
                 res = client.search(q)
                 self.assertEqual(1, res.total)
 
+                q2 = client.tagvals('tags')
```
Comment (ashtul) [inline test/test.py:622]: Let's add another document with additional words and see that we get the combined list.

**Review by ashtul — COMMENTED** [review]
Thank for the contribution. It looks great!
Just minor comments.

File: test/test.py [hunk 3]
```diff
@@ -596,8 +596,10 @@ def testTags(self):
             client.redis.flushdb()
             
             client.create_index((TextField('txt'), TagField('tags')))
+            
+            tags = 'foo,foo bar,hello;world'
 
-            client.add_document('doc1', txt = 'fooz barz', tags = 'foo,foo bar,hello;world')
+            client.add_document('doc1', txt = 'fooz barz', tags = tags)
```
Comment (rafie) [inline test/test.py:602]:
```
client.add_document('doc1', txt='fooz barz', tags=tags)
```

File: test/test.py (hunk 1 above)
Comment (maguec) [reply test/test.py]: Removed

File: test/test.py (hunk 3 above)
Comment (maguec) [reply test/test.py:603]: added another document to ensure more complete testing

**Review by ashtul — APPROVED** [review]
Looks great.
Thanks

**Pull request merged**